#return the point cloud of the respective cell
string cellID
---
sensor_msgs/PointCloud2 cloud
